### DISCUSSION TITLE
Have AddVegetarian AddVegan also in citizen quest? [changed]

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#2398] [Q&A] [answered]
Is it intentional that AddVegetarian and AddVegan are not in citizen achievement category, but only in their own category?

Is it impossible to put quest in more than one category or for some other reason?

**Comment by westnordost** [comment, marked as answer]
It is possible and it would make sense, yeah.

**Reply by westnordost** [reply]
done

**Reply by FloEdelmann** [reply]
@westnordost I think you might be able mark this discussion as "resolved".

**Reply by westnordost** [reply]
There is no such thing as marking a discussion as resolved. I can only delete it.

**Reply by FloEdelmann** [reply]
What does the "answered" filter on the discussions page do then?

https://github.com/streetcomplete/StreetComplete/discussions?discussions_q=is%3Aanswered

At the moment, all discussions are shown as "unanswered".

**Reply by matkoniecz** [reply]
I switched category to Q&A what allowed me to mark it as answered.

Also, note that "unanswered" is also empty: https://github.com/streetcomplete/StreetComplete/discussions?discussions_q=is%3Aunanswered

**Reply by smichel17** [reply]
It seems like outside the Q&A category, no discussions are shown as answered *or* unanswered:
https://github.com/streetcomplete/StreetComplete/discussions?discussions_q=is%3Aunanswered
